src/main/java/sx/blah/discord/util/RequestBuffer.java

```diff
@@ -58,7 +56,7 @@ public static void request(IVoidRequest request) {
 	 * @return The number of incomplete requests.
 	 */
 	public static int getIncompleteRequestCount() {
-		return incompleteRequestCount;
+		return requests.size();
```

Comment:
This is not enough. That gets the size of the bucket-requests map. But there could be more than one request queued per bucket.